.travis.yml

```diff
@@ -2,9 +2,13 @@ dist: trusty
 language: node_js
 node_js:
   - 'lts/*'
-install:
-  - npm i -g npm@6.1.0
-  - npm ci
+git:
+  depth: false
```

Comment:
This removes depth limit of 50 commits, so whole history is checked out. Let's maybe set it to 1?

Reply:
Yeah, that's me misreading the docs thinking that `false` would just check out one level. Seems like `1` would disable concurrent builds so I'll go with `5` so we've got something low that's not going to interfere with multiple PRs happening at the same time.